content/docs/1.4.0/volumes-and-nodes/expansion.md

```diff
@@ -109,3 +112,22 @@ If a volume is reverted to a snapshot with smaller size, the frontend of the vol
     xfs_growfs <the mount directory>
     umount /dev/longhorn/<volume name>
     ```
+
+#### Encrypted volume
+
+Due to [the upstream limitation](https://kubernetes.io/blog/2022/09/21/kubernetes-1-25-use-secrets-while-expanding-csi-volumes-on-node-alpha/), Longhorn cannot handle **online** expansion for encrypted volumes automatically unless you enable the feature gate `CSINodeExpandSecret`.
+
+If you cannot enable it but still prefer to do online expansion, you can:
+1. Login the node host the encrypted volume is attached to.
+2. Execute `cryptsetup resize <volume name>`. The passphrase this command requires is the field `CRYPTO_KEY_VALUE` of the corresponding secret.
+3. Expand the filesystem.
+
+#### RWX volume
+
+Currently, Longhorn is unable to expand the filesystem (NFS) for RWX volumes. - If you decide to expand a RWX volume manually, you can:
+
+1. Expand the block device of the RWX volume via PVC or UI.
+2. Figure out the share manager pod of the RWX volume then execute the filesystem expansion command. The share manager pod is typically named as `share-manager-<volume name>`.
+    ```shell
+    kubectl -n longhorn-system exec -it <the share manager pod> -- resize2fs /dev/longhorn/<volume name>
```

Comment:
Need to add `xfs_growfs` for XFS.

Reply:
Longhorn share manager pods are using `ext4`, should we add xfs?
https://github.com/longhorn/longhorn-share-manager/blob/e3ec44a91fa302f0280de6b93fddcd9525ef3bd8/app/cmd/server.go#L40

Reply:
My fault. You are right. The doc is correct!